src/components/ScreenWrapper/index.native.js

```diff
@@ -0,0 +1,30 @@
+import {React, useEffect} from 'react';
+import DevMenu from 'react-native-dev-menu';
+import compose from '../../libs/compose';
+import toggleTestToolsModal from '../../libs/actions/TestTool';
+import {propTypes, defaultProps} from './propTypes';
+import BaseScreenWrapper from './BaseScreenWrapper';
+import withEnvironment from '../withEnvironment';
+import withWindowDimensions from '../withWindowDimensions';
+
+const ScreenWrapper = (props) => {
+    useEffect(() => {
```

Comment:
I don't think we allow hooks. Can we please change this to class component and put it under `componentDidMount`?